wormhole-connect/src/utils/routes/bridge.ts

```diff
@@ -226,13 +226,21 @@ export class BridgeRoute extends RouteAbstract {
     return txId;
   }
 
-  async redeem(
-    destChain: ChainName | ChainId,
-    vaa: Uint8Array,
-    payer: string,
-  ): Promise<string> {
+  public async readyForRedeem(
+    txData: ParsedMessage | ParsedRelayerMessage,
+  ): Promise<boolean> {
+    const parsedVaa = await fetchVaa(txData);
```

Comment:
I noticed that the change introduced 4 calls to re-fetch the vaa.

Perhaps the method can be changed to receive an object from a union type (e.g: `type MessageInfo = VaaInfo | CCTPAttestationInfo`) and leave it to the route to pick which one to use. What do you think?